cursorless-talon/src/tutorial.py

```diff
@@ -0,0 +1,80 @@
+import json
+import re
+from pathlib import Path
+from typing import Callable
+
+import yaml
+from talon import actions, app
+
+from .get_action_spoken_form import lookup_action
+
+regex = re.compile(r"\{(\w+):([^}]+)\}")
+tutorial_dir = Path(
+    "/Users/pokey/src/cursorless-vscode/src/test/suite/fixtures/recorded/tutorial/unit-2-basic-coding"
+)
```

Comment:
this obviously won't generalize 😄. I would be tempted to let the extension own the yaml tbh